### ISSUE TITLE
Do kpoints and kpoints_distance need to exclusively set?

### ISSUE THREAD

**Issue by unkcpz** [aiidateam/aiida-quantumespresso#821]
https://github.com/aiidateam/aiida-quantumespresso/blob/3ede0053100f2617d32f45649e88cd8bef3cbfc6/src/aiida_quantumespresso/workflows/pw/base.py#L266-L289

As shown in the kpoints validator of `PwBaseWorkChain`, I think it makes more sense only one of them can be set?
I happened to set both of them without exception raise which lead to the spurious results in the calculation.

**Comment by sphuber** [comment]
If there are others that think it is better to be explicit and raise when both are defined, then I am happy to change the behavior. @mbercx ?

**Comment by sphuber** [comment]
I agree. Think a warning is better than failing and should be sufficient
